custom/icds_reports/utils/aggregation_helpers/distributed/ccs_record_monthly.py

```diff
@@ -245,7 +249,7 @@ def aggregation_query(self):
             ('child_name', 'case_list.child_name'),
             ('husband_name', 'person_cases.husband_name'),
             ('lmp', 'case_list.lmp'),
-            ('migration_status', 'person_cases.migration_status'),
+            ('migration_status', '{}'.format(migration_status)),
```

Comment:
can you try to replace this with?
```
('migration_status', 'COALESCE({},0)'.format(migration_status)),
```
I think this will allow us get rid of `0/NULL` test case failures even.

Reply:
okay

Reply:
done